exercises/modules/modules2.cairo

```diff
@@ -0,0 +1,56 @@
+// I AM NOT DONE
+// These modules have some issues, can you fix?
+// Run `starklings hint modules2` or `hint` watch command for a hint.
+
+use debug::PrintTrait;
+const YEAR: felt252 = 2050;
+
+mod order {
+    #[derive(Copy, Drop)]
+    struct Order {
+        name: felt252,
+        year: felt252,
+        made_by_phone: bool,
+        made_by_email: bool,
+        item: felt252,
+    }
+
+    fn new_order( name: felt252, made_by_phone: bool, item: felt252 ) -> Order {
+        Order {
+            name,
+            year: YEAR,
+            made_by_phone,
+            made_by_email: ! made_by_phone,
+            item,
+        }
+    }
+}
+
+mod order_utils {
```

Comment:
Perhaps a `TODO: explain what you have to do in this file` would be nice to have

Reply:
For this one they should figure it out themselves... From errors they get.